app/views/partners/_partner_row.html.erb

```diff
@@ -25,7 +25,11 @@
   <td>
     <% case status %>
     <% when "uninvited" %>
-      <%= invite_button_to(invite_partner_path(partner_row), confirm: "Send an invitation to #{partner_row.name} to begin using the partner application?") %>
+      <% if current_organization.single_step_invite_and_approve == false %>
```

Comment:
In general we want to have positive conditions. I'd swap the two sides of the condition and have it read `if current_organization.single_step_invite_and_approve?`